infrastructure/main.tf

```diff
@@ -199,6 +199,7 @@ module "claim-store-api" {
     FEATURE_TOGGLES_EMAILTOSTAFF = "${var.enable_staff_email}"
     FEATURE_TOGGLES_CCD_ENABLED = "${var.ccd_enabled}"
     FEATURE_TOGGLES_CCD_ASYNC_ENABLED = "${var.ccd_async_enabled}"
+    FEATURE_TOGGLES_SAVE_CLAIM_STATE_ENABLED = "${var.save_claim_state_enabled}"
```

Comment:
pls add this to cmc-integration-tests and helm values.yaml

Reply:
done